src/raiden_libs/logging.py

```diff
@@ -36,23 +36,30 @@ def setup_logging(log_level: str) -> None:
     )
 
 
-def change_bytes(event_dict: Dict, key: Any, val: bytes) -> Dict[str, Any]:
-    if len(val) == 20:
-        event_dict[key] = to_checksum_address(val)
-    else:
-        event_dict[key] = to_hex(val)
-    return event_dict
+def make_bytes_readable(value: Any) -> Any:
+    if isinstance(value, bytes):
+        if len(value) == 20:
+            return to_checksum_address(value)
+
+        return to_hex(value)
+    return value
+
+
+def apply_recursive(value: Any) -> Any:
+    if isinstance(value, list):
+        return [apply_recursive(x) for x in value]
```

Comment:
We should do the same for tuples, they are also quite common.

Reply:
Good, point. Added it.